api/proxy/.env.example

```diff
@@ -0,0 +1,90 @@
+# This .env.example file only describes the mandatory flags that must be set when running the EigenDA proxy.
+# All flags that are optional and have sane default values are omitted for brevity.
+# For a full list of flags, please refer to the output of `eigenda-proxy --help`,
+# which is also available at https://github.com/Layr-Labs/eigenda/blob/master/api/proxy/docs/help_out.txt.
+
+# === Storage Configuration ===
+
+# The storage backends to enable. Options are [V1, V2]
+EIGENDA_PROXY_STORAGE_BACKENDS_TO_ENABLE=V1,V2
+
+# Target EigenDA backend version (V1 or V2) for blob dispersals (POST routes).
+EIGENDA_PROXY_STORAGE_DISPERSAL_BACKEND=V2
```

Comment:
consider noting that whichever you choose, it must also be part of the `EIGENDA_PROXY_STORAGE_BACKENDS_TO_ENABLE` list

Reply:
[b91807f](https://github.com/Layr-Labs/eigenda/pull/1800/commits/b91807f2e8375cff57fcf8317d3284985e9ddfed)